controller/controller.go

```diff
@@ -100,27 +103,30 @@ func (ctrl *baseController) init(ctx context.Context) (*driver.Drivers, error) {
 		default:
 		}
 
-		log.Printf("[DEBUG] (ctrl) initializing task %q", task.Name)
+		taskName := task.Name()
+		log.Printf("[DEBUG] (ctrl) initializing task %q", taskName)
 		d, err := ctrl.newDriver(ctrl.conf, task, ctrl.watcher)
 		if err != nil {
 			return nil, err
 		}
 
-		err = d.InitTask(true)
-		if err != nil {
-			log.Printf("[ERR] (ctrl) error initializing task %q: %s", task.Name, err)
-			return nil, err
+		if task.IsEnabled() {
```

Comment:
ah sorry this was a half-baked idea that I should have removed since I wasn't planning to make any enable changes. Thanks for calling it out. I'll remove it!